doc/man3/DH_generate_parameters.pod

```diff
@@ -111,7 +111,7 @@ The parameter B<j> is invalid.
 
 =back
 
-DH_check_ex(), DH_check_params() and DH_check_pub_key_ex() are similiar with
+DH_check_ex(), DH_check_params() and DH_check_pub_key_ex() are similar to
 DH_check() and DH_check_params() respectively, but the error reasons are set
 to the OpenSSL error handling framework instead of returning by the function
 parameters.
```

Comment:
While your at it: would you mind correcting also the end of the sentence? IMHO it is grammatically not quite correct. One alternative formulation would be

>..., but the error reasons are added to the thread's error queue instead of returning them to the caller.